style.css

```diff
@@ -0,0 +1,288 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Barlow+Condensed:ital,wght@1,100&family=Poppins&display=swap');
+
+:root {
+  --blue: #6070ff;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
+  overflow-x: hidden;
+}
+
+/* header section */
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 20px;
+  background-color: white;
+  width: 100vw;
+}
+
+.logo,
+.menu {
+  color: var(--blue);
+  font-weight: 1000;
+  text-shadow: 0 3px 5px rgba(0, 0, 0, 0.338);
+  font-size: 1.4rem;
+}
+
+.headline_section {
+  background: url(/bg.png);
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  padding: 10px 20px 10px 50px;
+  border-radius: 0 0 20% 5%;
+  min-height: 70vh;
+  align-items: flex-start;
+  justify-content: center;
+  width: 100vw;
+}
+
+.headline_section h1 {
+  padding-bottom: 20px;
+  font-size: 3rem;
+}
+
+.headline_section p {
+  width: 70vw;
+}
+
+.headline_section span {
+  color: var(--blue);
+  padding: 10px 0;
+  font-weight: 500;
+}
+
+.headline_section span:hover {
+  cursor: pointer;
+  text-shadow: 0 2px 10px;
+}
+
+.headline_section ul i {
+  display: inline-flex;
+  padding-left: 5px;
+  font-size: 1.2em;
+}
+
+.headline_section ul i:hover {
+  cursor: pointer;
+  text-shadow: 0 2px 20px;
+}
+
+/* middle section */
+.works {
+  min-height: 70vh;
+  display: grid;
+  grid-template-columns: repeat(4/1fr);
+  align-items: center;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  min-height: 50vh;
+  border-radius: 10px;
+  margin-top: 20px;
+  width: 80vw;
+  align-items: flex-start;
+  padding: 30px;
+  justify-content: center;
+  gap: 10px;
+}
+
+.works .card img {
+  width: 100%;
+}
+
+.works .card .first_list {
+  display: flex;
+  list-style: none;
+  gap: 10px;
+}
+
+.works .card .second_list button {
+  background-color: #ebebff;
+  border: none;
+  padding: 5px 10px;
+  margin: 10px 0;
+  color: var(--blue);
+  border-radius: 12px;
+}
+
+.see_project {
+  width: 100px;
+  margin-top: 10px;
+  border: 1px solid var(--blue);
+  border-radius: 10px;
+  background-color: white;
+  color: var(--blue);
+  padding: 5px 0;
+}
+
+/* footer */
+footer {
+  display: grid;
+  grid-template-rows: repeat(3/1fr);
+  min-height: 60vh;
+  background-color: white;
+  border-radius: 5% 20% 5% 5%;
+  margin-top: 20px;
+  align-items: center;
+  justify-content: center;
+  padding: 10px 10vw;
+  gap: 20px;
+}
+
+footer .section1 .span {
+  color: var(--blue);
+  font-weight: 500;
+}
+
+.section1 {
+  padding: 30px 10px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.section2 {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.lang {
+  display: flex;
+  align-items: center;
+  background-color: #e5e5e5;
+  border-radius: 12px;
+  padding: 5px 10px;
+  font-weight: 500;
+  gap: 20px;
```

Comment:
i will start from here problems are that i am not able to align them properly,
suggestions: i can create it from scratch and then we will